src/api/app/views/webui/status_messages/new.html.haml

```diff
@@ -0,0 +1,24 @@
+- @pagetitle = 'New Status Message'
+.row
+  .col
+    .card
+      .card-body
```

Comment:
We should use the grid here.

So something like:
```
.col-12
  the title
.col-12
  the paragraph
.col-12.col-md-9.col-lg-6
  the form
```

Reply:
I think we tried that, but it totally broke the view. From the discussion about the "grid" I thought the grid was to stop using straight card, but instead using `.row > .col > .card` no?

Reply:
I think I understand now. I believe @dmarcoux suggests to add these columns inside the `card-body` not replacing what we had. I'm pushing a new commit changing this.

Reply:
@saraycp Yes, this is what I meant. The example from my previous comment should be done under `.card-body`, so starting on line 6. Sorry @vpereira, I wasn't clear enough!

Reply:
it was addressed!